core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java

```diff
@@ -273,6 +273,11 @@ protected ArtifactSignerProvider createArtifactSignerProvider(
 
           if (azureKeyVaultParameters.isAzureKeyVaultEnabled()) {
             LOG.info("Bulk loading keys from Azure key vault ... ");
+            /*
+             Note: Azure supports 25K bytes per secret. https://learn.microsoft.com/en-us/azure/key-vault/secrets/about-secrets
+             Each raw bls private key in hex format is approximately 100 bytes. We should store about 200 or fewer
+             `\n` delimited keys per secret.
+            */
```

Comment:
👍

Reply:
Thats the reason I added this comment, this is meant to be updated at https://docs.web3signer.consensys.net/how-to/use-signing-keys#bulk-load-consensus-layer-keys